src/components/PokeApp.js

```diff
@@ -67,35 +69,6 @@ function PokeApp() {
     }
   };
 
-  // get data for n number of moves for each Pokemon
-  const getPokemonMoveset = async (moves, n) => {
-    const moveSet = [];
-    // loop over a pokemon's moves array N=4 times
-    for (let i = 0; i < n; i += 1) {
-      const moveObj = moves[i];
-      const moveUrl = moveObj.move.url;
-      try {
-        const moveData = await fetchMove(moveUrl);
-        if (moveData) {
-          moveSet.push(moveData);
-        }
-      } catch (error) {
-        console.error(error);
-      }
-    }
-    // return a moveSet array
-    return moveSet;
-  };
-
-  const getMovesByPokemon = async (dexNo) => {
-    const moves = allPokemons.map((poke, i) => {
-      const moves = poke.moves;
-      // get 4 moves from all moves a pokemon can learn
-      const fourMoves = getPokemonMoveset(moves, 4);
-      return fourMoves;
-    });
-  };
-
```

Comment:
does this take care of all the fetching for moves once pokeapp first renders?

Reply:
So, those functions defined there aren't invoked(are unused) in that file, so I deleted them. Instead they are defined and invoked in another file(I believe it was `PokeCard`).